packages/manager/modules/nutanix/src/dashboard/general-info/controller.js

```diff
@@ -11,10 +11,17 @@ import { CLUSTER_STATUS } from '../../constants';
 
 export default class NutanixGeneralInfoCtrl {
   /* @ngInject */
-  constructor($translate, atInternet, ovhManagerRegionService, NutanixService) {
+  constructor(
+    $translate,
+    atInternet,
+    coreConfig,
+    ovhManagerRegionService,
+    NutanixService,
+  ) {
     this.$translate = $translate;
     this.atInternet = atInternet;
-    this.TRAVAUX_LINK = TRAVAUX_LINK;
+    this.TRAVAUX_LINK =
+      TRAVAUX_LINK[coreConfig.getUser()?.ovhSubsidiary] || TRAVAUX_LINK.DEFAULT;
```

Comment:
**nitpick:** Since the link must be updated for US Manager, you can use `coreConfig.getRegion()`
```suggestion
    this.TRAVAUX_LINK =
      TRAVAUX_LINK[coreConfig.getRegion()] || TRAVAUX_LINK.DEFAULT;
```